### PR TITLE
Add log info for disk attachment in block resize cases

### PR DESCRIPTION
**Pull request by lento-sun**
The disk attachment is a key step here. Without the info, it's
hard for debugger to manually check what happened according to
the log.

Signed-off-by: Yi Sun <yisun@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_blockresize.py [hunk 1]
```diff
@@ -148,7 +149,4 @@ def run(test, params, env):
                           "different from actual size from "
                           "'qemu-img info'")
     finally:
-        virsh.detach_disk(vm_name, target="vdd")
-
-        if os.path.exists(image_path):
-            os.remove(image_path)
```
Comment (dzhengfy) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blockresize.py]: Why is this line removed?

Comment (lento-sun) [reply libvirt/tests/src/virsh_cmd/domain/virsh_blockresize.py]: the image is generated into tmpdir, no need to recycle it within case, the framework will remove it. And I found sometimes the image removed before the "detach" actually finished, cause vm destroyed with error.
